Source/dom/__tests__/Archive.test.js

```diff
@@ -0,0 +1,17 @@
+/* globals expect, test */
+
+import { objectFromArchive } from '../wrapNativeObject'
+
+test('should archive object', () => {
+  const object = MSLayer.new()
```

Comment:
Probably want to create a Layer here: `const object = new Shape()` (and you need to import `Shape` obviously)